spec/tests/acceptance/acceptance_spec.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require 'webdrivers/chromedriver'
+require 'watir'
+require 'minitest/autorun'
+require_relative '../../spec_helper'
+
+describe 'Enter Job Query' do
+  before do
+    @browser ||= Watir::Browser.new :chrome
+  end
+
+  after do
+    @browser.close
+    # @headless.destroy
+  end
+
+  index_url = 'http://127.0.0.1:4001/'
+  valid_request = 'frontend engineer'
+  invalid_request = 'asdf'
+
+  it '(HAPPY) should be able to request a job query' do
+    # Given: index page
+    @browser.goto index_url
+
+    # When: user wants to send a request
+    # User is able to locate where to input and send query
+    @browser.text_field(name: 'query').present?
+    @browser.button(type: 'submit').present?
+    # Input a valid request
+    @browser.text_field(name: 'query').set(valid_request)
+    @browser.button(type: 'submit').click
+
+    # Then: user jumps to the correct details url
+    @browser.url.include? valid_request.sub! ' ', '+'
```

Comment:
violate the frozen string literal

Reply:
[Solved](https://github.com/KeroKnows/s-killer/pull/24/commits/230b33619fc6473ac0c9b60689d4a54264614c48)